### DISCUSSION TITLE
Any workaround to install Mu 1.1.0 or later on Raspberry Pi OS (Bullseye)?

### DISCUSSION THREAD

**Discussion by solzard** [mu-editor/mu#2396] [Q&A]
Mu on Raspi OS installed with Recommended Software or `apt install` is 1.0.3, which doesn't support the `Tidy` feature introduced in 1.1.0.

Installing as a Python module (`pip3 install mu-editor`) doesn't work on Raspi as [FAQ](https://codewith.mu/en/howto/1.2/install_with_python) clearly states:
> **Please note: this method does not currently work on Raspberry Pi**

Is there any hope to use the newer Mu on Raspi?

**Comment by hitsware** [comment]
PLUS ONE !
No RP2040 option on ' mode '  ....... I want to try ' PLOT '
For some reason doesn ' t even recognize Pico with circuit - Python
